.github/workflows/qe-tests.yml

```diff
@@ -106,6 +106,21 @@ jobs:
         - name: Set up PKI deployment by running playbook on "controller"
           run: vagrant provision controller
 
+        - name: Run CA related tests
+          run: |
+            vagrant ssh controller -c \
+            'pytest --ansible-host-pattern master \
+            --ansible-inventory /vagrant/inventory \
+            --ansible-module-path /vagrant/tests/dogtag/pytest-ansible/common-modules/ \
+            /vagrant/tests/dogtag/pytest-ansible/pytest/sanity/test_role_users.py \
+            --junit-xml pki_cert_junit.xml -qsvvv'
+            vagrant ssh controller -c \
+            'pytest --ansible-host-pattern master \
```

Comment:
Each `pytest` should run individually as a different GH action task. This will confuse the users when trying to debug the issue. IOW, when you combine 2 (or more) different pytests together, it will be difficult for the reader to understand/debug the issue.